### ISSUE TITLE
[3.12.0 RC] ACE Arsenal | Can't use enter to submit searches (Weapons/Uniforms search bar)

### ISSUE THREAD

**Issue by Joocebocks** [acemod/ACE3#5927]
**Arma 3 Version:** `Stable` (stable / rc / dev)
**CBA Version:** `stable` (stable / dev + commit hash)
**ACE3 Version:** `3.12.0 RC` (stable / dev + commit hash)

**Mods:**
```
- CBA_A3
- ace
- RHS (All mods)
- Cup Terrains
- 3CB (All Mods)
- TFAR
```

**Description:**
- Attempting to use enter to submit a search using the left most search bar for weapons and uniforms does not work. Pressing enter on the right most search bar for weapon attachments and magazines works fine.

**Steps to reproduce:**
- Add the ACE Arsenal to an object
- Access the arsenal
- Input a search using the top left most search bar
- Press enter to submit the search
- Note: Clicking the search button works fine, just couldn't use the enter key to submit

**Where did the issue occur?**
- Editor (Multiplayer)

**Placed Modules:**
- Will list if deemed relevant (numerous modules placed)

**RPT log file:**
- Will upload if actually needed.

NOTE: Submitting this issue away from my PC before I forgot. I checked out the Arsenal over the weekend and encountered the issue. Will test again with only ACE and CBA to confirm later today if needed.

**Comment by PabstMirror** [comment]
I couldn't reproduce this, maybe try again with 3.12 RC2
Both search boxes should work the same, so if the right one is working it probably isn't a keyboard issue...

**Comment by Joocebocks** [comment]
I realized the issue... The problem is numpad enter. I was searching for a weapon with a few numbers in its name and I used the numpad to enter those numbers and of course numpad enter to submit. I didn't reproduce it with the accessories section because I wasn't using the numpad to enter numbers for the search.

Enter/Return works fine but numpad enter is what isn't working. This is less than an issue and more of a QOL change that is not that pressing.
